modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -1,38 +1,127 @@
-= Controlling File Versions and Aging
+= Controlling File Versions
 
```

Comment:
pls add `:toc: right` direct underneath the title header

Reply:
👍

Reply:
you missed `right` as I wrote above...